src/sentry/static/sentry/app/components/events/meta/metaProxy.tsx

```diff
@@ -52,23 +56,25 @@ export class MetaProxy {
   }
 }
 
-export function withMeta(event) {
+export function withMeta<T>(event: T): T {
   if (!event) {
-    return null;
+    return event;
   }
 
   // Return unproxied `event` if browser does not support `Proxy`
   if (typeof window.Proxy === 'undefined' || typeof window.Reflect === 'undefined') {
     return event;
   }
 
-  const _meta = event._meta;
-  return new Proxy(event, new MetaProxy(_meta));
+  return new Proxy(event, new MetaProxy((event as any)._meta));
 }
 
-export function getMeta(obj, prop) {
-  if (typeof obj[GET_META] !== 'function') {
-    return null;
+export function getMeta<T extends {}>(
```

Comment:
```suggestion
export function getMeta<T extends {} | undefined>(
```

Reply:
I cannot do this or the type for prop will be broken. `obj` already allows for undefined